next-app/BUILD.bazel

```diff
@@ -0,0 +1,61 @@
+load("@npm//:next/package_json.bzl", next_bin = "bin")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+load("@aspect_rules_js//js:defs.bzl", "js_binary", "js_run_binary", "js_test")
+
+npm_link_all_packages(name = "node_modules")
+
+filegroup(
```

Comment:
nit: more one-sentence comments here might help someone new to Bazel to navigate

Reply:
👍